fdbserver/include/fdbserver/IClosable.h

```diff
@@ -0,0 +1,40 @@
+/*
+ * IClosable.h
```

Comment:
Nice interface. 

Should this be part of `fdbserver` package or `fdbclient`? Making it part of latter, it would have better usage. Thoughts?

Reply:
If someone need it, it can be moved in another PR. I'll keep it as is for now.